### ISSUE TITLE
nz-icon: aria-label is overwritten by nzType

### ISSUE THREAD

**Issue by tom-smu** [NG-ZORRO/ng-zorro-antd#9932]
### Reproduction link
[https://stackblitz.com/edit/wgyfn7qo-ycavvvzt](https://stackblitz.com/edit/wgyfn7qo-ycavvvzt)

### Steps to reproduce
1. Change aria-label at nz-icon
2. Inspect the element and notice, that the aria-label is still equal the nzType value

### What is expected?
The custom aria-label should be used

### What is actually happening?
The nzType is used as aria-label

| Environment | Info |
|---|---|
| ng-zorro-antd | 22.0.1 |
| Browser | every browser |
